kernel/reduction.ml

```diff
@@ -789,24 +789,6 @@ let infer_conv_leq ?(l2r=false) ?(evars=fun _ -> None) ?(ts=full_transparent_sta
     env univs t1 t2 = 
   infer_conv_universes CUMUL l2r evars ts env univs t1 t2
 
-(* This reference avoids always having to link C code with the kernel *)
```

Comment:
Is there a justification for this comment? What do we gain by not always linking C code? Can that affect e.g. JsCoq (cc @ejgallego).

Reply:
It is hard to infer the historical context as this has been there for a long time and the toolchain has changed quite a bit. In jscoq I do provide JS stubs for the C functions so I think that shouldn't matter there.

[In fact if someone would like to implement the stubs as either an Emscripten version of the C VM or as a native WebASM target vm_compute would happily work, see https://github.com/ejgallego/jscoq/issues/10]